### ISSUE TITLE
20% of reads get discarded in PE merging with Vsearch

### ISSUE THREAD

**Issue by VascoElbrecht** [VascoElbrecht/JAMP#26]
alignment score too low, or score drop too high, vsearch v2.19.0_macos_aarch64

Need to check setting changes

**Comment by OndroV** [comment]
In my experience [PEAR ](https://cme.h-its.org/exelixis/web/software/pear/doc.html) merged more reads in comparison with vsearch --fastq_mergepairs on the same data. Maybe something to consider?

**Comment by VascoElbrecht** [comment]
yes ^^ thank you for the suggestion. Looks like this is just how vsearch is, not a lot of options. Not happy about needing to use an additional program haha https://groups.google.com/g/vsearch-forum/c/ojqZYbEyUw4/m/9RMhPQbXAwAJ
